### PR TITLE
fix: wrap experiment properties under identity object

### PR DESCRIPTION
**Pull request by TCL735**
Follow-up to #4464 

Wraps the properties for the Credit 250 Experiment under an `identity` object before pushing to the `dataLayer`

### CODE REVIEW

File: src/shared/containers/GetOrganizations.tsx [hunk 1]
```diff
@@ -61,16 +64,31 @@ const GetOrganizations: FunctionComponent = () => {
...
+          account_type,
+          account_created_at,
+          id: meId,
+          email,
+          organization_id: orgId,
+        },
+      })
     }
-  }, [dispatch, me, quartzMeStatus])
+  }, [quartzMeStatus, status]) // eslint-disable-line react-hooks/exhaustive-deps
```
Comment (TCL735) [inline src/shared/containers/GetOrganizations.tsx:91]: If we use all of the dependencies suggested by the linter, then the `identity` gets pushed again to the `dataLayer` unnecessarily. We just want the status of the two api calls to be `RemoteDataState.Done` before executing the push.

File: src/shared/containers/GetOrganizations.tsx [hunk 2]
```diff
@@ -89,10 +107,10 @@ const GetOrganizations: FunctionComponent = () => {
               />
               <Route path="/orgs" component={App} />
               <Route exact path="/" component={RouteToOrg} />
-              {CLOUD && canAccessCheckout(me) && (
+              {CLOUD && canAccessCheckout(quartzMe) && (
```
Comment (hoorayimhelping) [inline src/shared/containers/GetOrganizations.tsx:110]: what's the difference between using `quartzMe` and `me` here?

**Review by hoorayimhelping — APPROVED** [review]

File: src/shared/containers/GetOrganizations.tsx (hunk 2 above)
Comment (TCL735) [reply src/shared/containers/GetOrganizations.tsx:110]: It has always been an object related to `quartzMe` but was previously named `me`. Yes, very confusing.

**Pull request merged**